### DISCUSSION TITLE
Documentação Swagger

### DISCUSSION THREAD

**Discussion by rabberto** [balta-io/2811#18] [Q&A] [answered]
Balta, bom dia, blz?
No curso Asp.Net 6, no vídeo referente a documentação da Api utilizando Swagger eu consegui fazer a configuração e funcionar corretamente como seu exemplo. Contudo na geração do Swagger não aparece a opção do "cadeado" indicando que o endpoint necessita de autenticação, bem como um local que após a autenticação e de posse do token, eu consiga liberar os endpoints.
Imagino que seja necessário acrescentar mais algumas configurações para que isso fique disponível.
Poderia me auxiliar, pfv?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @rabberto, vou tentar adicionar uma aula sobre isto em um futuro próximo... por enquanto, aqui está a doc do Swagger:
https://swagger.io/docs/specification/authentication/

Ai ele mostra como utilizar autenticação!

**Reply by rabberto** [reply]
Bom dia Balta!
Show de bola, muito obrigado, vou conferir hoje a noite a doc. pra tentar implementar no projeto aqui.
Abraço.
